trade-finance-manager-api/src/v1/validation/validate-payloads.ts

```diff
@@ -0,0 +1,34 @@
+import { asString } from '../../utils/validation';
+import { isValidMongoId } from './validateIds';
+
+export const validateUpdateUtilisationReportPayloadReport = (report: object) => {
+  if ('id' in report) {
+    const isValidReportId = isValidMongoId(asString(report.id)) && Object.keys(report).length === 1;
```

Comment:
Here our message is saying valid mongo id string but the use case also has no other parameters allowed apart from the id. Tbh I think I'd just remove the length check on the report keys as if there is a report id that's all we care about